sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/ServiceClient.cs

```diff
@@ -145,6 +145,15 @@ internal abstract class ServiceClient
 
         #endregion
 
+        #region Extract Summary
+
+        public abstract ExtractSummaryOperation StartExtractSummary(IEnumerable<string> documents, string language = default, ExtractSummaryOptions options = default, CancellationToken cancellationToken = default);
+        public abstract ExtractSummaryOperation StartExtractSummary(IEnumerable<TextDocumentInput> documents, ExtractSummaryOptions options = default, CancellationToken cancellationToken = default);
+        public abstract Task<ExtractSummaryOperation> StartExtractSummaryAsync(IEnumerable<string> documents, string language = default, ExtractSummaryOptions options = default, CancellationToken cancellationToken = default);
+        public abstract Task<ExtractSummaryOperation> StartExtractSummaryAsync(IEnumerable<TextDocumentInput> documents, ExtractSummaryOptions options = default, CancellationToken cancellationToken = default);
```

Comment:
Tip: these are already declared as virtual, so you might as well define them as virtual and throw here instead of in _LegacyServiceClient.cs_. Then just override in _LanguageServiceClient.cs_, which you have to anyway. Makes things easier moving forward.

Reply:
Done. I had to move the `private readonly TextAnalyticsClientOptions _options` member variable in both `LegacyServiceClient` and `LanguageServiceClient` classes to being a protected property on the parent `ServiceClient` class. I only cleaned up the methods for extractive summarization for now, but I will make this change to the rest of the methods in the `ServiceClient` class in a future PR.